build.sbt

```diff
@@ -40,7 +40,7 @@ val commonSettings = Seq(
     "org.mockito" % "mockito-core" % "2.18.0" % Test,
     "org.scalamock" %% "scalamock" % "5.1.0" % Test,
   ),
-  dependencyOverrides += "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.14.3",
+  dependencyOverrides += "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.17.2",
```

Comment:
@andrew-nowak  - doing this for all modules hasn't seemed to cause any problems AFAICT - but do you think it's safer to only do so for `usage`

Reply:
Nah, if it works for everything then let's apply it everywhere 👍